EXPLAINER.md

```diff
@@ -0,0 +1,227 @@
+
+
+### 1. What's part of the publication?
+
+- The [Web Application Manifest spec](https://w3c.github.io/manifest/) uses `scope` to define the extent of a web application. 
+
+- Every component of an EPUB publication must be listed in an XML package file.
+
+- Web sites do not explicitly define their boundaries.
+
+- XML sometimes uses the idea of *transclusion*, where the contents of other documents can be incorporated into a parent document based on hypertext references. This concept can also be realized in HTML using iframes, HTML imports, custom elements, etc. The working group has shown no interest in this approach. 
+
+```html
+<html>
+  <head>
+    <title>Moby-Dick</title>
+  </head>
+  <body>
+    <web-publication>
+      <iframe src="c001.html">
+      <iframe src="c002.html">
+      <iframe src="c003.html">
+    </web-publication>
+  </body>
+</html>
+
+```
+
+The web publications spec has essentially adopted EPUB's approach, with an explicit list of resources. Anything that is not part of the `resources` or `readingOrder` manifest members is considered to be outside the web publication. 
+
+### 2. Sequence of primary resources
+
+- HTML can express ordering of resources via `rel=prev` and `rel=next`, but these do not have UI support from the browsers (with notably rare exceptions).
+
+- HTML can also express an ordered list of links with the `nav` element. 
+
+- Transclusion can also define an ordering of the transcluded components. 
+
+- EPUB uses the `spine` element in the XML package file to determine a primary reading order.
+
+Once again, web publications use an explicit list outside of the content itself, via the `readingOrder` member of the manifest. 
+
+### 3. Metadata
+
+- EPUB includes metadata in the XML package file
+
+- There are many ways of embedding metadata in individual HTML files
+
+- Metadata about a web application can be expressed in the web application manifest file. 
+
+- With transclusion, one could define metadata in the parent file to apply to the whole.
+
+Web publication metadata is expressed in the manifest, using a vocabulary largely taken from schema.org. 
+
+
+
+
+### Relationship to Web Application Manifest
+
+The Web Publication Manifest appears to be very similar to the Web Application Manifest. Both are JSON files that are linked to from HTML, and provide metadata about a composite resource. Several [arguments](https://github.com/w3c/wpub/wiki/Options-for-Processing-a-Manifest) [have](https://github.com/w3c/wpub/issues/32) been made against using WAM.
+
+1. WP use cases are orthogonal to those of WAM. Nothing is stopping a creator of a web publication from also using a web application manifest, if the publication author desires for the publication to be installable, etc. 
+
+2. The web application manifest spec is not designed to be extensible. 
```

Comment:
That may not be 100% true, because they do talk about extension points, but it is very unclear how that would be used for an extension of processing.